docs/current_docs/api/chaining.mdx

```diff
@@ -92,19 +92,19 @@ Here is an example of listing the contents of a directory returned by a Dagger F
 
 ```shell
 git clone https://github.com/golang/example
-dagger -m github.com/kpenfound/dagger-modules/golang@v0.2.0 call build --source=./example/hello --args=. directory --path=. entries
+dagger -m github.com/kpenfound/dagger-modules/golang@v0.2.1 -c 'build . --source=./example/hello' | directory . | entries'
```

Comment:
Works differently from original, returns error `Error: json: error calling MarshalJSON for type *dagger.Directory: input: git.ref.tree.directory lstat /example/hello: no such file or directory`. Similar to another error, seems to not be referencing the host dir.

Reply:
For context, see:
- https://github.com/dagger/dagger/issues/9035